### DISCUSSION TITLE
Add Quarto Extension for AGU journals

### DISCUSSION THREAD

**Discussion by vonw** [quarto-dev/quarto-cli#4226] [Feature Requests]
Quarto, it would be great to add a journal extension for journals of the American Geophysical Union (AGU). @mikemahoney218 has already created one, so it should be easy to add to your website: https://github.com/mikemahoney218/quarto-agu. Please note that I think this could go a long way to extend the use of Quarto to the geophysical community. You might also consider developing an extension for European Geophysical Union (EGU) journal. Bottom line... Quarto rocks!

**Comment by mcanouil** [comment]
To add extensions to Quarto website, one has only to make a PR as described in <https://quarto.org/docs/extensions/>.

**Reply by cscheid** [reply]
(Thank you as always! I knew you'd know about that link and would respond faster than I'd learn :D)

**Reply by vonw** [reply]
Good point. I'll contact Mike Mahoney separately and encourage him to make PR. Thanks! Keep up the great work with Quarto.

**Reply by mcanouil** [reply]
Extension's authors are not necessarily the only ones that can make the PRs. 😅

**Comment by vonw** [comment]
Thanks. I’m obviously a newbie to PRs. But my intentions are good. I think Quarto is great and am hoping that folks in my area of research (geoscience) will find it useful.

**Comment by mcanouil** [comment]
Closing this as completed.
